internal/monitors/utilization/perfcounter/logical_disk_test.go

```diff
@@ -0,0 +1,203 @@
+package perfcounter
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/signalfx/golib/datapoint"
+	"github.com/signalfx/signalfx-agent/internal/monitors/telegraf/common/measurement"
+	"github.com/signalfx/signalfx-agent/internal/neotest"
+)
+
+func TestLogicalDiskMeasurement(t *testing.T) {
+	var monitorType = "signalfx-system-utilization"
+	var Measurement = "win_logical_disk"
+	var percentFreeSpace = float32(80)
+	var utilization = float64(100 - percentFreeSpace)
+	var freeMegabytes = float32(1024)
+	var usedBytes = ((float64(freeMegabytes) * megabytesToBytes) * 100 / (100 - utilization)) - (float64(freeMegabytes) * megabytesToBytes)
```

Comment:
If you just make `freeMegabytes` a `float64` to begin with there is no need for the conversions.  There is no need to micro-optimize stack usage in a unit test.

Reply:
It's float32 because that value is set in a Measurement further down and it's intended to match exactly what is passed into this code from the perf counter monitor.